src/components/CalendarMonth.jsx

```diff
@@ -52,16 +54,27 @@ const defaultProps = {
 export default class CalendarMonth extends React.Component {
   constructor(props) {
     super(props);
+
     this.state = {
-      weeks: getCalendarMonthWeeks(props.month, props.enableOutsideDays),
+      weeks: getCalendarMonthWeeks(
+        props.month,
+        props.enableOutsideDays,
+        props.firstDayOfWeek === null ? moment.localeData().firstDayOfWeek() : props.firstDayOfWeek,
```

Comment:
I do think you're right; that a value of `undefined` would trigger the default, and a value of `null` would not, so === null would work fine - but in general for style, yes. Not a huge deal, ofc.